src/Sarif.Multitool.FunctionalTests/TestData/UriMustBeAbsolute/Valid.sarif

```diff
@@ -38,7 +38,12 @@
         {
           "uri": "https://example.com/my-project"
         }
-      ]
+      ],
+      "properties": {
+        "expectedResults": {
+          "resultLocationPointers": []
```

Comment:
>resultLocationPointers [](start = 11, length = 22)

shouldn't this expected results object contain a rule id of some kind?

Reply:
I considered it. I didn't do it because I reasoned "all the results will have the same ruleId". But now you mention it, it's possible that a bug could cause the wrong rule to fire on the expected location. Ok, I'll do it in next PR.

---
In reply to: [216740798](https://github.com/Microsoft/sarif-sdk/pull/1081#discussion_r216740798) [](ancestors = 216740798)

Reply:
No, I don't need to do that. All I have to do is have the validate method ensure that each result has the ruleId for the rule under test. I'll show you in next PR.

---
In reply to: [216741872](https://github.com/Microsoft/sarif-sdk/pull/1081#discussion_r216741872) [](ancestors = 216741872,216740798)

Reply:
@michaelcfanning Turns out the "update" functionality isn't as easy as I thought. I'm going to let this percolate for a little while.